CHANGELOG.md

```diff
@@ -226,7 +225,10 @@ This project tries to follow [SemVer 2.0.0](https://semver.org/).
 
 - Changed versions of numerous dependencies:
 
+  <!--lint ignore maximum-line-length-->
+
```

Comment:
Is this scoped to the current indent level?

Reply:
Yes. Or rather, the following block.

`<!--lint disable` will disable until reenabled
`<!--lint ignore` will disable for only the next block

I was a bit surprised that it counted the entire list as one block.

Would changing to `disable`+`enable` make it clearer, or OK to leave it?

Reply:
https://github.com/remarkjs/remark-lint#ignore-warnings

Reply:
OK to leave imo 👍🏻